<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomDialog</class>
 <widget class="QWidget" name="CustomDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CustomDialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: qlineargradient(x1:0, y1:0, x1:1, y1:6, stop: 0.9#39393b, stop:1#49494b);
color:white;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,1,1,1,1" rowminimumheight="0,0,0,0,0">
     <item row="4" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="m_OkButton">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>32</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
color:rgb(21,21,21);
border: 1px solid rgb(94, 94, 94);
background-color:rgb(125,129,133);
min-height:30px;
border-radius:6px;
}

QPushButton::disabled{
 background-color: rgba(92, 93, 95, 80%);
 color: rgb(49, 49, 49);
 border: 1px solid rgb(59, 59, 59);
}

QPushButton:hover{
background-color:rgb(164, 168, 172);
}

QPushButton:pressed{
background-color:rgb(166, 166, 166);
}</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QLabel" name="m_Content">
       <property name="styleSheet">
        <string notr="true">background:transparent;</string>
       </property>
       <property name="text">
        <string>Content</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="m_Title">
       <property name="styleSheet">
        <string notr="true">background:transparent;
font-weight:bold;</string>
       </property>
       <property name="text">
        <string>Title</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="Line" name="line">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(99,99,99);</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="Line" name="line_2">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(99,99,99);</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
